dev015-making-apps-more-powerful/04-custom-data-final/add-custom-data/OpenExtensionsDemo.cs

```diff
@@ -1,34 +1,29 @@
-﻿using Microsoft.Identity.Client;
+﻿using device_code_flow_console;
+using Microsoft.Identity.Client;
 using Newtonsoft.Json.Linq;
 using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Net.Http;
 using System.Net.Http.Headers;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace add_custom_data
 {
-    class OpenExtensionsDemo
+    class OpenExtensionsDemo : PublicAppUsingDeviceCodeFlow
```

Comment:
OpenExtensionsDemo now inherits from PublicAppUsingDeviceCodeFlow, and therefore benefits from its methods